pkg/loop/internal/relayer/pluginprovider/chainreader/chain_reader.go

```diff
@@ -228,6 +243,20 @@ func (c *Server) GetLatestValue(ctx context.Context, request *pb.GetLatestValueR
 	return &pb.GetLatestValueReply{RetVal: encodedRetVal}, nil
 }
 
+func (c *Server) BatchGetLatestValue(ctx context.Context, pbRequest *pb.BatchGetLatestValueRequest) (*pb.BatchGetLatestValueReply, error) {
```

Comment:
Are there any limits to batch size? Should we consider them or leave that up to the implementations?

Reply:
I would leave that to the implementation. Right now the evm impl. uses defaults, but it if needed it can be added to interface level or to config